### PR TITLE
Update docs to use K8s manifest generator

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,8 +222,49 @@ We currently provide three options for launching Weave Scope in ECS:
 ## <a name="using-weave-scope-with-kubernetes"></a>Using Weave Scope with Kubernetes
 
 Scope comes with built-in Kubernetes support. We recommend to run Scope natively
-in your Kubernetes cluster using
-[these resource definitions](https://github.com/helm/charts/tree/master/weavescope/manifests).
+in your Kubernetes cluster using manifest generator service.
+
+The simples way to get latest release of Scope deployed on a Kubernetes cluster:
```
Comment (peterbourgon) [inline README.md]: simplest

File: README.md [hunk 2]
```diff
@@ -222,8 +222,49 @@ We currently provide three options for launching Weave Scope in ECS:
 ## <a name="using-weave-scope-with-kubernetes"></a>Using Weave Scope with Kubernetes
 
 Scope comes with built-in Kubernetes support. We recommend to run Scope natively
-in your Kubernetes cluster using
-[these resource definitions](https://github.com/helm/charts/tree/master/weavescope/manifests).
+in your Kubernetes cluster using manifest generator service.
```
Comment (peterbourgon) [inline README.md]: using the manifest generator service

**Comment by errordeveloper** [comment]
All typos fixed.

**Comment by errordeveloper** [comment]
As @abuehrle suggested, we are gonna make it part of #1416.

**Pull request closed without merging**